src/Stratis.Bitcoin.IntegrationTests/Wallet/WalletOperationsTests.cs

```diff
@@ -223,57 +223,6 @@ public async Task CreateWalletWith12WordsMnemonic()
             account.HdPath.Should().Be("m/44'/105'/0'");
         }
 
-        [Fact]
-        public async Task CreateWalletWith12WordsChineseMnemonic()
```

Comment:
This shouldn't be removed its a useful test

Reply:
But it failed. 
I'm ok to revert it or close whole PR if someone is willing to fix it and other tests

Reply:
We should revert this test as it's passing for me.